### PR TITLE
Propose: add preferWords option

### PR DESCRIPTION
**Pull request by Leko**
I added an option so that users could use the same terminology throughout the documents.
We can make this a fixable rule as we know the acceptable term.

This is not a breaking change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ textlint --rule @textlint-ja/no-synonyms README.md
      * 例) ["ウェブアプリ", "ウェブアプリケーション"]
      */
     allows?: string[];
+    /**
+     * 使用を許可する見出し語の配列
+     * 定義された見出し語以外の同義語をエラーにします
+     * 例) ["ユーザー"] // => 「ユーザー」だけ許可し「ユーザ」などはエラーにする
+     */
+    allowsOnly?: string[];
```
Comment (azu) [inline README.md]: another options?

- `preferWords`
- `dict`
- `myWords`

他のallowはtrueなら無視する感じなので、allowじゃない方が良い気がします。
あんまりいいものが思いついてないですが

Comment (Leko) [reply README.md]: `preferWords`いいですね！リネームします。

File: src/textlint-rule-no-synonyms.ts [hunk 2]
```diff
@@ -87,7 +95,19 @@ const report: TextlintRuleReporter<Options> = (context, options = {}) => {
                             allowAlphabet,
                             allowNumber
                         });
-                        if (items.length >= 2) {
+                        const allowedTerm = preferWords.find(midashi => itemGroup.getItem(midashi))
```
Comment (azu) [inline src/textlint-rule-no-synonyms.ts]:
```suggestion
                        const preferWord = preferWords.find(midashi => itemGroup.getItem(midashi))
```

**Review by azu — COMMENTED** [review]
Thanks for PR!

Almost LGTM

**Comment by azu** [comment]
https://github.com/textlint-ja/textlint-rule-no-synonyms/releases/tag/v1.2.0
Thanks! Release it

**Pull request merged**